src/index.ts

```diff
@@ -36,6 +36,7 @@ export {default as Overlay, OverlayTypes} from './components/overlay';
 export {default as RadioButton, RadioButtonPropTypes, RadioButtonProps} from './components/radioButton/RadioButton';
 export {default as RadioGroup, RadioGroupPropTypes, RadioGroupProps} from './components/radioButton/RadioGroup';
 export {default as Switch, SwitchProps} from './components/switch';
+export {default as TabController, TabControllerProps, TabBarItemProps} from './components/tabController';
```

Comment:
`TabBarItemProps` might conflict in the future with our TabBar component which will probably have also TabBatItemProps :/ 
Maybe we can name this one TabControllerItemProps, WDYT?

Reply:
I think it might get confusing since all the sub components there are (for TabController), and I'm not sure people won't need their props as well.
Maybe we can add them statically? Either on the component or create another "MegaProp" that has them all?
So either TabController.Props, TabController.TabBar.Props... or TabControllerProps.Main, TabControllerProps.TabBar...
Not sure how easy it is to do or understand, WDYT?

Reply:
I'm not sure you can create this type of nesting in typings... 
If it's possible then let's do it.. 

I guess that's the reason other libraries' typings have weird long names..

Reply:
I went with `TabControllerProps` namespace, let me know if that's ok

Reply:
The namespace is nice, but now users will need to do  this?

`TabControllerProps.TabControllerProps`? it's a little weird don't you think?

Reply:
:s

Any suggestions? `TabControllerProps.Props` or maybe create the `namespace` as `TabController` and everything will be inside it (but the it'll be `TabController.TabController` and worse `TabController.TabController.TabBar`).

Reply:
TBH, Im starting to think we should drop the namespace thing, it's a little messy and too nested. 
Im not sure people are used to import typings this way.
I prefer to have 
TabControllerProps
TabControllerBarProps
TabControllerItemProps
TabControllerPageProps, etc...

It's not the best, but feels simpler.. WDYT?

Reply:
I agree, changed as you recommended